src/client/components/pages/Faculty/FacultyPage.tsx

```diff
@@ -128,6 +138,18 @@ const FacultySchedule: FunctionComponent = (): ReactElement => {
       });
   }, [dispatchMessage, isStaleData, isInitialized, closeAbsenceModal]);
 
+  const filteredFaculty = useMemo(() => {
```

Comment:
Can you just add a brief docblock noting what this `useMemo` call is doing? (Even though it's mostly apparent from the code I do find it helps to have those docs when we're revisiting something like this in the future.